src/components/Form/Fields/DropdownField/DropdownField.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import '../styles.scss';
+
+import { InputFieldProps, Option } from '../types';
+
+const DropdownField = ({ field, value, onFieldChange }: InputFieldProps) => {
+  const [error, setError] = useState<string>('');
+
+  const validate = (value: string) => {
+    setError('');
+    if (!field.options || field.options.length === 0) {
+      setError(`${field.externalName} has no options available`);
+      return;
+    }
+    onFieldChange(field.internalName, value);
+  };
+
+  const selectValue = value || field.value || '';
+  return (
+    <div className="field">
```

Comment:
We should add a special dropdown class as we will need some dropdown specific stylings